cfpb/sbl-frontend
# Design Review - Errors & Warnings - Part 2

closes #538
closes #539 
closes #542
closes #551 

## More changes (05/21/2024)
- content(errors/warnings): error/warnings counts and table rows use commas (e.g. 1,000) 
- stylefix(table): hidden table borders are now present again on single/register fields
- content(errors/warnings): record/records changed to 'found'

## More changes (05/20/2024)
- style(success error/warnings alert): reduced margin-bottom to 30px (30px between success alert and Filing Nav Buttons)
- style(warnings): verify text limited to 670px, well limited to 770px
- style(warnings): multifield warnings now also has table borders
- feat(errors & warnings): Updated the count to be the rows affected by errors/warnings
- ~~content(errors/warnings): added 'record' or 'records' word to count statement~~

## Changes
- style(single/multi/register errors): 45px margin-bottom
- style(tables): only multifield errors tables have borders, 
- content(errors): removed bottom alerts
- style(errors/warnings - success): use negative margin (ensures 30px from alert to filing nav buttons)
- content(errors): "there is" instead of "there may be"
- style(warnings): restrict header to 670px
- enhancement(warnings): Reworked warnings alert logic

## How to test (dev)
The only functionality changes are in:

- enhancement(warnings): Reworked warnings alert logic -- see `FilingWarningsAlerts.tsx`
- feat(errors & warnings): Updated the count to be the rows affected by errors/warnings -- see `getRecordsAffected` function.

## Note
- Once deployed, @natalia-fitzgerald can verify
